### PR TITLE
[RDR] Optimise cluster for RDR dialog fails with error msg

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
BZ 4.15: https://bugzilla.redhat.com/show_bug.cgi?id=2257222

### CODE REVIEW

File: packages/odf/constants/data-protection.ts [hunk 1]
```diff
@@ -1,6 +1,10 @@
 export const DISASTER_RECOVERY_TARGET_ANNOTATION =
   'ocs.openshift.io/clusterIsDisasterRecoveryTarget';
 
+// "~1" is used to represent a "/", else any "patch" call will treat prefix as a path
+export const DISASTER_RECOVERY_TARGET_ANNOTATION_WO_SLASH =
+  'ocs.openshift.io~1clusterIsDisasterRecoveryTarget';
+
```
Comment (SanjalKatiyar) [inline packages/odf/constants/data-protection.ts:7]: when we create StorageSystem, then also we might want to fix this ??

Comment (SanjalKatiyar) [reply packages/odf/constants/data-protection.ts:7]: https://github.com/red-hat-storage/odf-console/blob/master/packages/odf/components/utils/common.ts#L486

Comment (GowthamShanmugam) [reply packages/odf/constants/data-protection.ts:7]: Initial deployment is working fine from 4.14, only path based update is a problem

File: packages/ocs/dashboards/persistent-internal/status-card/osd-migration/osd-migration-progress.tsx [hunk 2]
```diff
@@ -60,78 +57,76 @@ export const OSDMigrationProgress: React.FC<OSDMigrationProgressProps> = ({
   return (
     <>
       {migrationStatus !== OSDMigrationStatus.PENDING && <Divider />}
-      <HealthBody>
```
[the remark this answers is not in the document]
Comment (GowthamShanmugam) [reply packages/ocs/dashboards/persistent-internal/status-card/osd-migration/osd-migration-progress.tsx:63]: I moved OSDMigrationProgress component inside HealthBody on status-card.ts

**Pull request merged**